lib/backend/shadowbackend/client.go

```diff
@@ -82,6 +86,39 @@ func NewClient(config Config, authConfRaw interface{}, stats tally.Scope) (*Clie
 	}, nil
 }
 
+func extractAuthConfigs(config Config, masterAuthConfig backend.AuthConfig) (interface{}, interface{}, error) {
```

Comment:
Are you planning to add unit tests in the same PR for these newly added methods ?

Reply:
I've covered them fully now.